.gitignore

```diff
@@ -25,6 +25,14 @@ wheels/
 .installed.cfg
 *.egg
 
+# OS generated files #
+######################
+.DS_Store?
```

Comment:
shouldn't be enough with a .DS_Store entry?

Reply:
Going to replace this by githubs .gitignore template for macos https://github.com/github/gitignore/blob/master/Global/macOS.gitignore